### PR TITLE
completed problems of day 5, 6, 7

### PR DESCRIPTION
**Pull request by raghavsrm**

### CODE REVIEW

File: src/com/raghav/javabasics/day5/MultiplyTwoMatrices.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    public void multiplyMatrices(int[][] a, int[][] b) {
+        if(a[0].length != 2 || a[1].length!= 2 || b[0].length != 2 || b[1].length != 2) {
+            System.out.println("Enter 2 by 2 matrix");
+            return;
+        }
+        int c[][] = new int[2][2];
+        for(int i=0; i<b.length; i++) {
+            for (int j = 0; j < b.length; j++) {
+                c[i][j] = 0;
+            }
```
Comment (mspranav) [inline src/com/raghav/javabasics/day5/MultiplyTwoMatrices.java:22]: Not Needed as default is ```0```

File: src/com/raghav/javabasics/day6/ConvertArrayOfStringToArrayList.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    private void convertToArrayList(String[] str) {
+        if (str == null || str.length == 0 ) {
+            System.out.println("Array is Empty/Null");
+            return;
+        }
+        ArrayList<String> newStr = new ArrayList();
+        for(int i=0; i<str.length;i++){
+            newStr.add(i, str[i]);
```
Comment (mspranav) [inline src/com/raghav/javabasics/day6/ConvertArrayOfStringToArrayList.java:20]: it appends directly. Include index only if you want to place the element into a specific index

File: src/com/raghav/javabasics/day6/FindDuplicatesInArrayList.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+// Write a program to identify duplicates in the ArrayList.
+import java.util.Arrays;
+import java.util.List;
+
+public class FindDuplicatesInArrayList {
+    public static void main(String args[]){
+        FindDuplicatesInArrayList result = new FindDuplicatesInArrayList();
+        result.findDuplicatesInArrayList(Arrays.asList(new String[]{"world","Hello","world","Hello", "world"}));
+    }
+    public void findDuplicatesInArrayList(List arrList){
```
Comment (mspranav) [inline src/com/raghav/javabasics/day6/FindDuplicatesInArrayList.java:11]: Can change this to accept ```String[]```

File: src/com/raghav/javabasics/day7/CompareReturnCommonElementsArrayList.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        set.addAll(arrayList);
+        arrayList.addAll(set);
+        if (set == null || set.size() == 0) {
+            System.out.println("No common elements!!!");
+            return;
+        }
+        for (int i : set) {
+            System.out.print(i + " ");
+        }
+    }
```
Comment (mspranav) [inline src/com/raghav/javabasics/day7/CompareReturnCommonElementsArrayList.java:45]: Not Necessary as you already found the intersection of the two ArrayLists in the first loop

File: src/com/raghav/javabasics/day7/CompareReturnNonCommonElementsArrayList.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            if (!arraylist.contains(integer)) {
+                System.out.print(integer + " ");
+            }
+        }
+        for(int i = 0; i < arrList2.size(); i++) {
+            if(!arraylist.contains(arrList2.get(i))) {
+                System.out.print(arrList2.get(i)+" ");
+            }
+        }
+    }
```
Comment (mspranav) [inline src/com/raghav/javabasics/day7/CompareReturnNonCommonElementsArrayList.java:36]: Can add Not ```!``` for the first loop and print the ones that satisfy the condition

**Review by mspranav — CHANGES_REQUESTED** [review]
Please review the comments and make changes accordingly

File: src/com/raghav/javabasics/day7/CompareReturnNonCommonElementsArrayList.java [hunk 6]
```diff
@@ -6,32 +6,25 @@
...
+            if (!arrList2.contains(integer)) {
                 arraylist.add(integer);
             }
         }
         for (Integer integer : arrList2) {
-            if (arrList1.contains(integer)) {
+            if (!arrList1.contains(integer)) {
                 arraylist.add(integer);
             }
         }
```
Comment (mspranav) [inline src/com/raghav/javabasics/day7/CompareReturnNonCommonElementsArrayList.java]: Not necessary for the same reason as you removed (a code block) the intersection of two arrays.

**Review by mspranav — DISMISSED** [review]
Please do let me know if you did not understand my review comments

File: src/com/raghav/javabasics/day7/CompareReturnNonCommonElementsArrayList.java (hunk 6 above)
Comment (raghavsrm) [reply src/com/raghav/javabasics/day7/CompareReturnNonCommonElementsArrayList.java]: Two loops are necessary to get the non-repeating elements from the 2 arrayLists.

**Review by shashanksurya — APPROVED** [review]
Approved

**Pull request closed without merging**